### DISCUSSION TITLE
Can't duplicate tabs

### DISCUSSION THREAD

**Discussion by marioekaputta** [bahasa-ai/otomatis-guidance#18] [General]
When I duplicate tabs on the dev platform, I must reload it as well otherwise it won's showed
![image](https://user-images.githubusercontent.com/71392755/129302497-6f1c56c0-77ff-4be5-9b49-a3da1a8485e0.png)

**Comment by mgilangjanuar** [comment]
Front-end dev be like:

![giphy](https://user-images.githubusercontent.com/3146378/129303484-9702db16-8bf5-446b-9381-5a024bb8b835.gif)

**Comment by nurulai** [comment]
perasaan kejadiannya biasanya pas edit authentication.. kok ini d action yaa wkwk

**Comment by jundialwan** [comment]
@marioekaputta you still face this issue?

I think we already merged fixes related to this issue.

**Reply by marioekaputta** [reply]
@jundialwan no I'm no longer face this issue. Solved
